test/util/util.go

```diff
@@ -264,3 +266,37 @@ func GetEnrollmentIdFromText(text string) string {
 	}
 	return ""
 }
+
+// RandString generates a random string of length 'n' using lowercase alphabetic characters.
+func RandString(n int) (string, error) {
+	const alphanum = "abcdefghijklmnopqrstuvwxyz"
+	var bytes = make([]byte, n)
+	if _, err := rand.Read(bytes); err != nil {
+		return "", fmt.Errorf("failed to generate random string: %w", err)
+	}
+	for i, b := range bytes {
+		bytes[i] = alphanum[b%byte(len(alphanum))]
+	}
+	return string(bytes), nil
+}
+
+// GenerateTimestamps returns start of current and next year in RFC3339 format.
+func GenerateTimestamps() (string, string) {
```

Comment:
The name of the function doesn't  reflect the actual intent of the function.
I would suggest something like GenerateYearStartEnd() or GetCurrentYearBounds()